src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -469,16 +474,17 @@ function _createGeckoThread(extraMarkers = []): GeckoThread {
       schema: {
         stack: 0,
         time: 1,
-        responsiveness: 2,
```

Comment:
I changed this fixture to the new eventDelay while I'm here.

Reply:
But is the data correct? Well I see it's all "0" here anyway so it's probably not interesting...